src/main/scala/com/typesafe/sbt/ProguardKeys.scala

```diff
@@ -0,0 +1,86 @@
+package com.typesafe.sbt
+
+import com.typesafe.sbt.SbtProguard.autoImport.ProguardOptions.Filtered
+import com.typesafe.sbt.proguard.Merge
+import com.typesafe.sbt.proguard.Merge.Strategy
+import sbt.{File, SettingKey, TaskKey}
+
+trait ProguardKeys {
+  val proguardVersion = SettingKey[String]("proguard-version")
+  val proguardDirectory = SettingKey[File]("proguard-directory")
+  val proguardConfiguration = SettingKey[File]("proguard-configuration")
+  val binaryDeps = TaskKey[Seq[File]]("binaryDeps")
```

Comment:
The key should have `proguard` prefixed, as `proguardBinaryDeps`, and use lower cased `taskKey[Seq[File]]("...")`.
http://www.scala-sbt.org/1.x/docs/Plugins-Best-Practices.html#Avoid+namespace+clashes

The same for rest of the keys.